packages/model/addon/-private/model.js

```diff
@@ -2413,7 +2413,7 @@ if (DEBUG) {
         until: '5.0',
         since: { available: '4.8', enabled: '4.8' },
       });
-      return originalReopen.call(this, arguments);
```

Comment:
🤦🏻 

Can we use apply instead? IIRC it passes through arguments when proxied like this more cleanly.

Reply:
going to merge this, we can always change it later. It's deprecation code anyway.